http-client-core/src/main/java/io/micronaut/http/client/HttpClientConfiguration.java

```diff
@@ -369,6 +379,8 @@ public void setChannelOptions(Map<String, Object> channelOptions) {
     }
 
     /**
+     * [not in java.net HttpClient].
```

Comment:
should not we support this for Java net http Client as well? https://www.baeldung.com/java-httpclient-timeout

Reply:
Agreed, fixed in https://github.com/micronaut-projects/micronaut-core/pull/8441/commits/eac7562a4c0cefe86abd7e1961cc2696b67b0894

and 

https://github.com/micronaut-projects/micronaut-core/pull/8777/commits/5db898f2d6d8004e03fd4236bae21b7d6ffa2515